go/client/cmd_login.go

```diff
@@ -165,6 +191,19 @@ func (c *CmdLogin) Cancel() error {
 	return nil
 }
 
+func (c *CmdLogin) readPaperKeyFromStdin() (string, error) {
```

Comment:
use the technique we use in `cmd_oneshot` and maybe factor out that function to use in both places.

Reply:
Cool, I wanted to ask about it but I started this early in the morning. The stdin stuff seemed really messy considering that it sometimes went into interactive mode.